### ISSUE TITLE
Custom command syntax

### ISSUE THREAD

**Issue by eurochriskelly** [fgeorges/mlproj-core#14]
Currently, the format for custom commands involves prepending a colon, e.g.
```
commands: {
   ':mycommand': () => { /*stuff*/ }
}
mlproj :mycommand
```

This means I have to quote my command names in which is ok when the config is json but less so in JS.

Maybe it would be nice, and easy to remember,  to follow the `npm run` style and allow commands without the colon. e.g.:

```
commands: {
   mycommand: () => { /*stuff*/ }
}
mlproj run mycommand
```

**Comment by fgeorges** [comment]
Yeah, makes sense.  The ":" was a simple convention to avoid clashes with future commands.  Using `run` solves that also, probably in a better way.

**Comment by eurochriskelly** [comment]
Also, running `mlproj run` should list all available custom commands, just as `npm run` does!

**Comment by fgeorges** [comment]
Fixed in core 0.26.1.
